samples/display/cfb_custom_font/CMakeLists.txt

```diff
@@ -9,4 +9,8 @@ project(cfb_custom_font)
 target_sources(app PRIVATE src/main.c)
 
 set(gen_dir ${ZEPHYR_BINARY_DIR}/include/generated/)
+if(CONFIG_DISPLAY_MCUX_ELCDIF)
+generate_cfb_font_for_target(app fonts/8x8text_darkGrayNoShadow.png ${gen_dir}/cfb_font_dice.h 8 8 --first 49 --last 54)
```

Comment:
While the mono version of this has previously been merged in, extending this example to include basic latin characters would perhaps make this more useful, rather than the current range of `49-54`, corresponding to `1-6` on the ASCII table. There is a need for small fonts with clear license terms, so having a clearly licensed 5x7 or 8x8 fixed-width font in RGB565 and monochrome would be useful here.

[Current font options](https://github.com/zephyrproject-rtos/zephyr/blob/master/subsys/fb/cfb_fonts.c) are on the large side at 10x16, 15x24 and 20x32, so having an 8x8 version in this demo would address this issue.

Reply:
the font is edited from https://opengameart.org/content/8x8-font, as the customer font demos 1-6, so I only add 1-6 in the png

Reply:
NP. I'll try to get a larger character in as a separate PR using this method, as well as finding an appropriately licensed 5x7 font.

Reply:
I add subsys/fb/font_8x8.c from Linux kernel fonts, which is a full set